### PR TITLE
Move tx-related parts from wallet service to tx wallet service

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/services/tx-wallet.ts [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+import { CellIsNotYetLive, TransactionIsNotCommittedYet } from 'exceptions/dao'
+import TypeConvert from 'types/type-convert'
+import FeeMode from 'models/fee-mode'
+import DaoUtils from 'models/dao-utils'
+import TransactionSize from 'models/transaction-size'
+import TransactionFee from 'models/transaction-fee'
+import logger from 'utils/logger'
+import Keychain from 'models/keys/keychain'
+
+export default class TxWalletService {
```
Comment (ashchan) [inline packages/neuron-wallet/src/services/tx-wallet.ts]: `TxWallet` is confusing. Please consider a better name such as `TransactionService`.

Comment (classicalliu) [reply packages/neuron-wallet/src/services/tx-wallet.ts]: A class already named `TransactionService`, how about `TransactionSender` ? It's a class combine transaction and wallet (to sign transactions).

Comment (ashchan) [reply packages/neuron-wallet/src/services/tx-wallet.ts]: `TransactionSender` looks good to me 👍

Comment (classicalliu) [reply packages/neuron-wallet/src/services/tx-wallet.ts]: updated

File: packages/neuron-wallet/src/services/transaction-sender.ts [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+import TransactionFee from 'models/transaction-fee'
+import logger from 'utils/logger'
+import Keychain from 'models/keys/keychain'
+
+export default class TransactionSender {
+  private static instance: TransactionSender
+
+  private walletService: WalletService
+
+  public static getInstance = () => {
```
Comment (ashchan) [inline packages/neuron-wallet/src/services/transaction-sender.ts]: I feel singleton is unnecessary for this class.

Comment (classicalliu) [reply packages/neuron-wallet/src/services/transaction-sender.ts]: Removed

**Pull request merged**